GirafIntegrationTest/tests/test_department_controller.py

```diff
@@ -373,9 +396,10 @@ def test_department_can_remove_pictogram_from_department_with_department(self):
         NOTE: this endpoint is deprecated
         """
         response = delete(f'{BASE_URL}v1/Department/resource/{pictograms["cyclopean"]["id"]}',
-                          headers=auth(department_token)).json()
-        self.assertTrue(response['success'])
-        self.assertEqual(response['errorKey'], 'NoError')
+                          headers=auth(department_token))
+        response_body = response.json()
```

Comment:
Is this assignment needed when it's not used? Also applies to the next three tests.